packages/unuse/package.json

```diff
@@ -27,7 +27,8 @@
     "dist"
   ],
   "dependencies": {
-    "alien-signals": "^2.0.5"
+    "alien-signals": "^2.0.5",
```

Comment:
Isn't it weird to have both of them here?

Reply:
I can do a stacked PR if you like to

Reply:
Or nevermind, I will just add unEffect in a separate PR (parallel, not stacked) and then rebase this one. As we need unEffect anyway later on.